src/library_syscall.js

```diff
@@ -1532,4 +1449,45 @@ for (var x in SyscallsLibrary) {
   wrapSyscallFunction(x, SyscallsLibrary, false);
 }
 
+function unimplementedSycall(name) {
+  SyscallsLibrary[name + '__nothrow'] = true;
+  SyscallsLibrary[name + '__proxy'] = false;
+  SyscallsLibrary[name + '__unimplemented'] = true;
+  msg = `err('warning: unsupported syscall: ${name}');\n`;
+  if (SyscallsLibrary[name]) {
+    SyscallsLibrary[name] = modifyFunction(SyscallsLibrary[name].toString(), (name, args, body) => {
+            return `function(${args}) { ${msg}\n ${body}\n }\n`;
+          });
+  } else {
+    errcode = cDefine('ENOSYS');
+    SyscallsLibrary[name] = `function() { ${msg}; return -${errcode}; }`;
+  }
```

Comment:
The `msg` is only added to existing functions. Is that to keep the current behavior? Seems reasonable, but I think it should also be commented on here, as reading the code, my initial reaction is that it's a bug not to have the message on line 1463 as well.

Reply:
The message is added in both cases.. no?   Its added to existing functions on line 1459 and included in new functions on line 1463.

Reply:
Oh, oops, I misread that, sorry...